### DISCUSSION TITLE
Daily 13/12/2023

### DISCUSSION THREAD

**Discussion by lucasdpt** [Sinan-Karakaya/R-Type#59] [Daily]
C'est l'anniversaire de mon frère aujourd'hui

**Comment by lucasdpt** [comment]
Implémentation de la lib réseau dans le serveur

**Reply by jovanHillion** [reply]
Hello ajd c'est du scripting :)

**Comment by Sinan-Karakaya** [comment]
Ajd je fix le loading de scene, puis je rejoins jovan sur le scripting

**Comment by JasonLAGOUTE** [comment]
Hello, aujourd'hui je vais être avec Lucas pour implémenter la lib réseau dans le serveur.
